### PR TITLE
Adding private sendTx function

### PR DESCRIPTION
**Pull request by chatton**
## Description

closes: #2239
closes: #2241

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/apps/27-interchain-accounts/controller/keeper/msg_server.go [hunk 1]
```diff
@@ -62,7 +62,8 @@ func (s msgServer) SendTx(goCtx context.Context, msg *types.MsgSendTx) (*types.M
 		return nil, sdkerrors.Wrap(channeltypes.ErrChannelCapabilityNotFound, "module does not own channel capability")
 	}
 
-	seq, err := s.Keeper.SendTx(ctx, chanCap, msg.ConnectionId, portID, msg.PacketData, msg.TimeoutTimestamp)
+	absoluteTimeout := uint64(ctx.BlockTime().UnixNano()) + msg.TimeoutTimestamp
+	seq, err := s.sendTx(ctx, chanCap, msg.ConnectionId, portID, msg.PacketData, absoluteTimeout)
```
Comment (chatton) [inline modules/apps/27-interchain-accounts/controller/keeper/msg_server.go:66]: I'm not 100% clear if there is more required than this change with the private function.

File: modules/apps/27-interchain-accounts/controller/keeper/relay.go [hunk 2]
```diff
@@ -17,6 +17,10 @@ import (
 // absolute timeoutTimestamp must be provided. If the packet is timed out, the channel will be closed.
 // In the case of channel closure, a new channel may be reopened to reconnect to the host chain.
 func (k Keeper) SendTx(ctx sdk.Context, _ *capabilitytypes.Capability, connectionID, portID string, icaPacketData icatypes.InterchainAccountPacketData, timeoutTimestamp uint64) (uint64, error) {
+	return k.sendTx(ctx, nil, connectionID, portID, icaPacketData, timeoutTimestamp)
+}
+
+func (k Keeper) sendTx(ctx sdk.Context, _ *capabilitytypes.Capability, connectionID, portID string, icaPacketData icatypes.InterchainAccountPacketData, timeoutTimestamp uint64) (uint64, error) {
```
Comment (colin-axner) [inline modules/apps/27-interchain-accounts/controller/keeper/relay.go:23]:
```suggestion
func (k Keeper) sendTx(ctx sdk.Context, connectionID, portID string, icaPacketData icatypes.InterchainAccountPacketData, timeoutTimestamp uint64) (uint64, error) {
```
we can remove the channel capability from the private func API

**Review by colin-axner — APPROVED** [review]
Nice!

**Review by charleenfei — APPROVED** [review]
lgtm!

**Pull request merged**